README.md

```diff
@@ -1,6 +1,6 @@
 # ![Foxbox Client](https://raw.githubusercontent.com/fxbox/app/master/app/img/icons/32.png "Foxbox Client") Foxbox Client
 
-> A client for Foxbox
+> A web client for Foxbox (with support for emulating the server component in the meantime).
```

Comment:
Which parts of the server component does it emulate? You mean the fact that the `gulp` command starts a web server with this app's static content on `http://localhost:8000/`, or is there more?